about.html

```diff
@@ -0,0 +1,5 @@
+---
+layout: page
+title: About
+permalink: /about/
```

Comment:
"permalink" property isn't required if the name of the .html file is the same as the URL you want in the end. :-)